tests/_data/mu-plugins/slr-test.php

```diff
@@ -77,6 +77,15 @@ function () {
 			\WP_CLI::success( 'Transients cleaned.' );
 		}
 	);
+	
+	\WP_CLI::add_command(
+		'slr:auth-token',
```

Comment:
I would have called this `slr:get-auth-token`.
What is this for?

Reply:
I was using POSTMan to check the SAAS API locally, and that requires the auth token. So, I was using this and thought might be helpful for others as well.

Will update the name.